editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/infopanel/DescriptionPanelFragment.kt

```diff
@@ -0,0 +1,57 @@
+package ca.ubc.eml.soiltopargraphy.editor.ui.infopanel
+
+import android.arch.lifecycle.ViewModelProviders
+import android.graphics.ColorSpace
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+
+import ca.ubc.eml.soiltopargraphy.editor.R
+
+/**
+ * Fragment where user enters Flag name and description
+ */
+
+class DescriptionPanelFragment : Fragment() {
+
+    private var mViewModel: DescriptionPanelViewModel? = null
```

Comment:
```suggestion
    private lateinit var mViewModel: DescriptionPanelViewModel
```
Tip: you can use the `lateinit` keyword for variables that you set once, so you don't need the null checks later.